torchci/pages/kpis.tsx

```diff
@@ -136,14 +136,25 @@ export default function Kpis() {
           title={"viable/strict lag (Daily)"}
           queryName={"strict_lag_historical"}
           queryCollection={"pytorch_dev_infra_kpis"}
-          queryParams={[...timeParams]}
+          queryParams={{
+            ...clickhouseTimeParams,
+            // Missing data prior to 2024-10-01 due to migration to ClickHouse
+            ...(startTime < dayjs("2024-10-01") && {
+              startTime: dayjs("2024-10-01")
+                .utc()
+                .format("YYYY-MM-DDTHH:mm:ss.SSS"),
+            }),
+            granularity: "week",
```

Comment:
you overwrite the granularity here, feature or bug?

Reply:
bug, fixed